### PR TITLE
[USERINIT] Add unattend.inf functionality to livecd

### PR DESCRIPTION
**Pull request by learn-more**
Allows booting straight to desktop

### CODE REVIEW

File: base/system/userinit/livecd.c [hunk 1]
```diff
@@ -805,13 +834,59 @@ StartDlgProc(
...
+        pUnattend->LocaleID = wcstol(Buffer, NULL, 16);
+    }
+}
 
 VOID
 RunLiveCD(
     PSTATE pState)
 {
+    LIVECD_UNATTEND Unattend = {0};
+    WCHAR UnattendInf[MAX_PATH];
```
Comment (HBelusca) [inline base/system/userinit/livecd.c:881]:
```suggestion
    WCHAR UnattendInf[MAX_PATH];

```

File: base/system/userinit/livecd.c [hunk 2]
```diff
@@ -11,6 +11,14 @@ HWND hList;
 HWND hLocaleList;
 BOOL bSpain = FALSE;
 
+typedef struct _LIVECD_UNATTEND
+{
+    BOOL bEnabled;
+
+    LCID LocaleID;
```
Comment (HBelusca) [inline base/system/userinit/livecd.c:18]:
```suggestion
    BOOL bEnabled;
    LCID LocaleID;
```

File: base/system/userinit/userinit.h [hunk 3]
```diff
@@ -58,6 +58,7 @@ typedef struct
     PAGESTATE NextPage;
     RUN Run;
     IMGINFO ImageInfo;
+    struct _LIVECD_UNATTEND* Unattend;
```
Comment (HBelusca) [inline base/system/userinit/userinit.h:61]: Why using a pointer instead of just an embedded structure as member?

Comment (learn-more) [reply base/system/userinit/userinit.h:61]: Didnt want to dump too much crap in userinit.h

**Pull request merged**